docs/content/en/docs/installation/helm/_index.md

```diff
@@ -5,7 +5,31 @@ weight: 15
 ---
 
 The Aiven Operator for Kubernetes can be installed via [Helm](https://helm.sh/). 
-Please follow The instruction in the [aiven-charts](https://github.com/aiven/aiven-charts) GitHub repository.
+
+First add the [aiven helm repository](https://github.com/aiven/aiven-charts):
+
+```bash
+helm repo add aiven https://aiven.github.io/aiven-charts && helm repo update
+```
+
+Next install the custom resource definitions:
+
+```bash
+helm install aiven-operator-crds aiven/aiven-operator-crds
+```
+
+Finally install the operator:
+
+```bash
+helm install aiven-operator aiven/aiven-operator
+```
```

Comment:
Add an instruction to verify the installation, you can use `$ helm status aiven-operator` for example.

Reply:
:heavy_check_mark: